src/main/webapp/javascript/recipe-rendering.js

```diff
@@ -14,20 +14,19 @@ createRecipeCard = (divID, recipeInfo) => {
 
   const id = recipeInfo["id"];
   const name = recipeInfo["name"];
-  const idToken = getIdToken();
 
-  const addToPlannerButton = createElement("button", " Planner", {"class": "card-button action-button bottom more-left fas fa-plus add-to-planner-btn"});
-  addToPlannerButton.addEventListener('click', () => manageList("add", id, name, idToken, 'planner'));
+  const addToPlannerButton = createElement("button", " Planner", {"class": "card-button action-button bottom more-left add-to-planner-btn fas fa-plus"});
+  addToPlannerButton.addEventListener('click', () => manageList("add", id, name, "planner"));
 
-  const addToCookbookButton = createElement("button", " Cookbook", {"class": "card-button action-button bottom more-right fas fa-plus add-to-cookbook-btn"});
-  addToCookbookButton.addEventListener('click', () => manageList("add", id, name, idToken, 'cookbook'));
+  const addToCookbookButton = createElement("button", " Cookbook", {"class": "card-button action-button bottom more-right add-to-cookbook-btn fas fa-plus"});
+  addToCookbookButton.addEventListener('click', () => manageList("add", id, name, "cookbook"));
 
   const removeFromPlannerButton = createElement("button", " Planner ", {"class": "card-button action-button bottom more-left fa fa-remove remove-from-planner-btn"});
-  removeFromPlannerButton.addEventListener('click', () => manageList("remove", id, name, idToken, 'planner'));
+  removeFromPlannerButton.addEventListener('click', () => manageList("remove", id, name, 'planner'));
```

Comment:
nit: change 'planner' to use "" to be consistent with above. Same for 'cookbook' below